diff --git a/node_modules/abstract-socket/build/Makefile b/node_modules/abstract-socket/build/Makefile
index 7b98bd6..857db80 100644
--- a/node_modules/abstract-socket/build/Makefile
+++ b/node_modules/abstract-socket/build/Makefile
@@ -308,8 +308,8 @@ ifeq ($(strip $(foreach prefix,$(NO_LOAD),\
 endif
 
 quiet_cmd_regen_makefile = ACTION Regenerating $@
-cmd_regen_makefile = cd $(srcdir); /home/harisvsulaiman/.npm-packages/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py -fmake --ignore-environment "--toplevel-dir=." -I/tmp/tmp-18663ARfoYjM6jPbj/node_modules/abstract-socket/build/config.gypi -I/home/harisvsulaiman/.npm-packages/lib/node_modules/npm/node_modules/node-gyp/addon.gypi -I/home/harisvsulaiman/.node-gyp/8.0.0/include/node/common.gypi "--depth=." "-Goutput_dir=." "--generator-output=build" "-Dlibrary=shared_library" "-Dvisibility=default" "-Dnode_root_dir=/home/harisvsulaiman/.node-gyp/8.0.0" "-Dnode_gyp_dir=/home/harisvsulaiman/.npm-packages/lib/node_modules/npm/node_modules/node-gyp" "-Dnode_lib_file=node.lib" "-Dmodule_root_dir=/tmp/tmp-18663ARfoYjM6jPbj/node_modules/abstract-socket" "-Dnode_engine=v8" binding.gyp
-Makefile: $(srcdir)/../../../../home/harisvsulaiman/.node-gyp/8.0.0/include/node/common.gypi $(srcdir)/build/config.gypi $(srcdir)/binding.gyp $(srcdir)/../../../../home/harisvsulaiman/.npm-packages/lib/node_modules/npm/node_modules/node-gyp/addon.gypi
+cmd_regen_makefile = cd $(srcdir); /usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py -fmake --ignore-environment "--toplevel-dir=." -I/home/harisvsulaiman/work/push/pushy/node_modules/abstract-socket/build/config.gypi -I/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi -I/home/harisvsulaiman/.node-gyp/8.0.0/include/node/common.gypi "--depth=." "-Goutput_dir=." "--generator-output=build" "-Dlibrary=shared_library" "-Dvisibility=default" "-Dnode_root_dir=/home/harisvsulaiman/.node-gyp/8.0.0" "-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp" "-Dnode_lib_file=node.lib" "-Dmodule_root_dir=/home/harisvsulaiman/work/push/pushy/node_modules/abstract-socket" "-Dnode_engine=v8" binding.gyp
+Makefile: $(srcdir)/../../../../../../../usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi $(srcdir)/build/config.gypi $(srcdir)/binding.gyp $(srcdir)/../../../../../.node-gyp/8.0.0/include/node/common.gypi
 	$(call do_cmd,regen_makefile)
 
 # "all" is a concatenation of the "all" targets from all the included
diff --git a/node_modules/abstract-socket/build/config.gypi b/node_modules/abstract-socket/build/config.gypi
index 377eb50..2f39640 100644
--- a/node_modules/abstract-socket/build/config.gypi
+++ b/node_modules/abstract-socket/build/config.gypi
@@ -64,105 +64,18 @@
     "nodedir": "/home/harisvsulaiman/.node-gyp/8.0.0",
     "copy_dev_lib": "true",
     "standalone_static_library": 1,
-    "cache_lock_stale": "60000",
-    "ham_it_up": "",
-    "legacy_bundling": "",
-    "sign_git_tag": "",
-    "user_agent": "npm/4.6.1 node/v8.0.0 linux x64",
-    "always_auth": "",
-    "bin_links": "true",
-    "key": "",
-    "description": "true",
-    "fetch_retries": "2",
-    "heading": "npm",
+    "user_agent": "yarn/1.0.2 npm/? node/v8.0.0 linux x64",
     "init_version": "1.0.0",
-    "user": "1000",
-    "allow_same_version": "",
-    "if_present": "",
-    "force": "",
-    "only": "",
-    "cache_min": "10",
-    "init_license": "ISC",
-    "editor": "vi",
-    "rollback": "true",
-    "tag_version_prefix": "v",
-    "cache_max": "Infinity",
-    "userconfig": "/home/harisvsulaiman/.npmrc",
-    "timing": "",
-    "tmp": "/tmp",
-    "engine_strict": "",
-    "init_author_name": "",
-    "init_author_url": "",
-    "depth": "Infinity",
-    "save_dev": "",
-    "usage": "",
-    "metrics_registry": "https://registry.npmjs.org/",
-    "progress": "true",
-    "https_proxy": "",
-    "sso_type": "oauth",
-    "onload_script": "",
-    "rebuild_bundle": "true",
-    "shell": "/usr/bin/zsh",
-    "save_bundle": "",
+    "init_license": "MIT",
+    "version_tag_prefix": "v",
     "prefix": "/home/harisvsulaiman/.npm-packages",
-    "dry_run": "",
-    "scope": "",
-    "cache_lock_wait": "10000",
-    "registry": "https://registry.npmjs.org/",
-    "browser": "",
-    "save_optional": "",
-    "searchopts": "",
-    "versions": "",
-    "cache": "/home/harisvsulaiman/.npm",
-    "send_metrics": "",
-    "global_style": "",
+    "registry": "https://registry.yarnpkg.com",
     "ignore_scripts": "",
-    "version": "",
-    "viewer": "man",
-    "local_address": "",
-    "color": "true",
-    "fetch_retry_mintimeout": "10000",
-    "maxsockets": "50",
-    "sso_poll_frequency": "500",
-    "umask": "0002",
-    "fetch_retry_maxtimeout": "60000",
-    "logs_max": "10",
-    "message": "%s",
-    "ca": "",
-    "cert": "",
-    "global": "",
-    "link": "",
-    "unicode": "true",
-    "access": "",
-    "also": "",
-    "save": "",
-    "searchlimit": "20",
-    "unsafe_perm": "true",
-    "long": "",
-    "production": "",
-    "auth_type": "legacy",
-    "node_version": "8.0.0",
-    "tag": "latest",
-    "git_tag_version": "true",
-    "shrinkwrap": "true",
-    "fetch_retry_factor": "10",
-    "proprietary_attribs": "true",
+    "version_git_message": "v%s",
+    "version_git_tag": "true",
+    "version_git_sign": "",
     "strict_ssl": "true",
-    "save_exact": "",
-    "globalconfig": "/home/harisvsulaiman/.npm-packages/etc/npmrc",
-    "init_module": "/home/harisvsulaiman/.npm-init.js",
-    "dev": "",
-    "parseable": "",
-    "globalignorefile": "/home/harisvsulaiman/.npm-packages/etc/npmignore",
-    "cache_lock_retries": "10",
-    "searchstaleness": "900",
     "save_prefix": "^",
-    "scripts_prepend_node_path": "warn-only",
-    "group": "1000",
-    "init_author_email": "",
-    "searchexclude": "",
-    "git": "git",
-    "optional": "true",
-    "json": ""
+    "ignore_optional": ""
   }
 }
diff --git a/node_modules/abstract-socket/lib/abstract_socket.js b/node_modules/abstract-socket/lib/abstract_socket.js
index a243486..3b7809e 100644
--- a/node_modules/abstract-socket/lib/abstract_socket.js
+++ b/node_modules/abstract-socket/lib/abstract_socket.js
@@ -1,7 +1,7 @@
 'use strict';
 
 const net = require('net');
-const binding = require('bindings')('abstract_socket.node');
+const binding = require('../build/Release/abstract_socket.node');
 
 const socket  = binding.socket;
 const bind    = binding.bind;
